### ISSUE TITLE
Improve disk setup with services

### ISSUE THREAD

**Issue by afbjorklund** [afbjorklund/buildroot4kubernetes#9]
Currently requires using fdisk and mount/resize2fs manually.

Would be nice to (safely) avoid this, and do automatic setup.

**Comment by afbjorklund** [comment]
Could at least document the current setup procedure.

Nothing special, but a bit tedious and not user-friendly

**Comment by afbjorklund** [comment]
```console
# fdisk /dev/vda

Welcome to fdisk (util-linux 2.36.2).
Changes will remain in memory only, until you decide to write them.
Be careful before using the write command.


Command (m for help): p
Disk /dev/vda: 20 GiB, 21474836480 bytes, 41943040 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
Disklabel type: dos
Disk identifier: 0x00000000

Device     Boot Start     End Sectors  Size Id Type
/dev/vda1         264 1331463 1331200  650M 83 Linux

Command (m for help): d
Selected partition 1
Partition 1 has been deleted.

Command (m for help): c
DOS Compatibility flag is set (DEPRECATED!)

Command (m for help): n
Partition type
   p   primary (0 primary, 0 extended, 4 free)
   e   extended (container for logical partitions)
Select (default p): p
Partition number (1-4, default 1): 1
First sector (22-41943039, default 22): 264
Last sector, +/-sectors or +/-size{K,M,G,T,P} (264-41943039, default 41943039): 

Created a new partition 1 of type 'Linux' and of size 20 GiB.
Partition #1 contains a ext4 signature.

Do you want to remove the signature? [Y]es/[N]o: n

Command (m for help): w

The partition table has been altered.
Syncing disks.

# resize2fs /dev/vda1 
resize2fs 1.45.6 (20-Mar-2020)
Filesystem at /dev/vda1 is mounted on /; on-line resizing required
old_desc_blocks = 1, new_desc_blocks = 2
The filesystem on /dev/vda1 is now 5242847 (4k) blocks long.

# df -h /
Filesystem                Size      Used Available Use% Mounted on
/dev/root                19.7G    611.8M     18.3G   3% /
```

**Comment by afbjorklund** [comment]
Fixed, 8864d8f4f72e87cf11d9ce7b467807d478779adf
